experimenter/experimenter/changelog/views.py

```diff
@@ -1 +1,30 @@
-# Create your views here.
+from django.shortcuts import get_object_or_404
+from django.views.generic import ListView
+
+from experimenter.experiments.models import NimbusChangeLog, NimbusExperiment
+
+
+class NimbusChangeLogsView(ListView):
+    model = NimbusChangeLog
+    template_name = "changelog/test.html"
+    context_object_name = "experiment"
+
+    def get(self, request, *args, **kwargs):
+        self.object = self.get_object()
+        return super().get(request, *args, **kwargs)
+
```

Comment:
here- The `get` method is unnecessary in this case since it only calls the parent class's get method with the same arguments. You can remove this method altogether, and the parent class's get method will be called automatically.